### PR TITLE
Utter fixes

### PR DESCRIPTION
**Pull request by wochinge**
**Proposed changes**:
- make 'text' the first positional argument in `utter_message`
- fix `utter_elements`
- return empty list instead of None for buttons + elements

**Status (please check what you already did)**:
- [x] made PR ready for code review
- [x] added some tests for the functionality
- [x] updated the documentation
- [ ] updated the changelog
- [x] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa-sdk#code-style) for instructions)

### CODE REVIEW

**Comment by wochinge** [comment]
@erohmensing will merge this as soon as you merged yours (the test will fail otherwise :-D). also updating the changelog then

File: rasa_sdk/executor.py [hunk 1]
```diff
@@ -26,21 +26,21 @@ def __init__(self) -> None:
 
     def utter_message(
         self,
+        text: Optional[Text] = None,
         image: Optional[Text] = None,
         json_message: Dict[Text, Any] = None,
```
Comment (erohmensing) [inline rasa_sdk/executor.py]:
```suggestion
        json_message: Optional[Dict[Text, Any]] = None,
```

File: rasa_sdk/executor.py [hunk 2]
```diff
@@ -26,21 +26,21 @@ def __init__(self) -> None:
...
     ) -> None:
         """"Send a text to the output channel"""
 
         message = {
             "text": text,
-            "buttons": buttons,
-            "elements": elements,
+            "buttons": buttons or [],
+            "elements": elements or [],
             "custom": json_message,
```
Comment (erohmensing) [inline rasa_sdk/executor.py]:
```suggestion
            "custom": json_message or {},
```
Not sure

Comment (wochinge) [reply rasa_sdk/executor.py]: Do we need this? I mean we actually don't even need it for the lists 👀

Comment (wochinge) [reply rasa_sdk/executor.py]: will add it

Comment (erohmensing) [reply rasa_sdk/executor.py]: yeah i was wondering if we needed it for the lists as well haha

**Pull request merged**